test/test_algorithm.py

```diff
@@ -107,7 +107,7 @@ def _missing_upload_file(
     try:
         caught_exception = False
         spec = tator.models.AlgorithmManifestSpec(name='test.yaml', upload_url='not_there')
-        response = tator_api.save_algorithm_manifest(name=basename, algorithm_manifest_spec=spec)
+        tator_api.save_algorithm_manifest(project = project, algorithm_manifest_spec=spec)
```

Comment:
I think this is right - I don't think the response var was used here and `save_algorithm_manifest` should take a project arg.